apps/web/components/eventtype/AssignmentWarningDialog.tsx

```diff
@@ -0,0 +1,39 @@
+import { useRouter } from "next/navigation";
+import type { Dispatch, SetStateAction } from "react";
+import type { MutableRefObject } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Dialog, ConfirmationDialogContent } from "@calcom/ui";
+
+interface IAssignmentWarningDialog {
```

Comment:
if it is an interface we don't need to prefix it with "I", so we can simply call it `AssignmentWarningDialogProps` or just `Props` since its straight forward that they are props for the component.